srcpkgs/psuinfo/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'psuinfo'
+pkgname=psuinfo
+version=1.0
+revision=1
+noarch=yes
+depends="python3>=3.5 python3-psutil"
+short_desc="A command to display customizable system info it Tint2 or other panels"
+maintainer="Piotr Miller <nwg.piotr@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/nwg-piotr/psuinfo"
+distfiles="https://github.com/nwg-piotr/psuinfo/archive/v${version}.tar.gz"
+checksum=3b26b17e3da5b5f736b1c1235bc2411c315b7cb32a82616a7c3c8e30e16687e9
+
+do_install() {
+	vmkdir usr/bin
+	vmkdir usr/share/${pkgname}
+	vmkdir usr/share/tint2
+	vcopy "icons/*.*" usr/share/${pkgname}
+	vcopy "psuinfo-1.0-2" usr/share/${pkgname}/${pkgname}
+	vcopy "configs/*.*" usr/share/tint2
+	ln -sf /usr/share/psuinfo/psuinfo ${DESTDIR}/usr/bin
+}
```

Comment:
I don't think there's an explizit rule about it but most templates have a newline at the end of the file, please add one here.

Reply:
Right, I must have deleted it by accident. Thank you.